Vagrantfile

```diff
@@ -209,6 +209,7 @@ def build_binary_with_pyinstaller(boxname)
     . borg-env/bin/activate
     cd borg
     pyinstaller --clean --distpath=/vagrant/borg scripts/borg.exe.spec
+    tar -czvf borg-macosx64.tar.gz --strip-components=2 dist/borg-macosx64
```

Comment:
maybe use `borg-dir` here, so it works for all platforms? see also comments below.

Reply:
Removed this again, so it can be done outside the VM.

> maybe use borg-dir here, so it works for all platforms? see also comments below.
It's not needed for any other platform and it's very confusing for the user to get a directory with 20 library files. I wouldn't publish it for every platform.

Reply:
iirc we had also cases on other platforms. iirc when wanting to use capabilities on linux, one can only work with an executable, not a script. and some people also complained about unpacking time on linux.

so i think we should offer this for all platforms and let people choose what they want.

best would be a single tar file that includes both builds, so I do not need to sign and upload more files than currently.

Reply:
Then we build both for everyone. Just needs a small adjustment to the PyInstaller spec file.